.github/workflows/automatic-updates.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           script: |
             const { default: script } = await import(`${process.env.GITHUB_WORKSPACE}/build-automation.mjs`);
-            await script(github);
+            return script(github);
```

Comment:
Is the await not needed so it doesn't return before getting the result?

Reply:
the script always awaits the result, and since we want to use the returned value, we don't need a `return await`